entities-viewings-collector/src/test/scala/io/renku/entities/viewings/search/ProjectQuerySpec.scala

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright 2023 Swiss Data Science Center (SDSC)
+ * A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
+ * Eidgenössische Technische Hochschule Zürich (ETHZ).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.renku.entities.viewings.search
+
+import io.renku.entities.search.model.{MatchingScore, Entity => SearchEntity}
+import io.renku.entities.viewings.search.RecentEntitiesFinder.Criteria
+import io.renku.generators.Generators.Implicits._
+import io.renku.http.server.security.model.AuthUser
+
+import java.time.Instant
+
+class ProjectQuerySpec extends SearchTestBase {
+
+  it should "return one project entry if viewed multiple times" in {
+    val project = renkuProjectEntities(visibilityPublic)
+      .withActivities(activityEntities(stepPlanEntities()))
```

Comment:
Personally, for the sake of simplicity, I'd maybe not add activities or datasets for these specs.

Reply:
I think I can remove the datasets here instead. I thought first, why not add them, just to also see if they are not returned. But can also remove